extensions/ql-vscode/src/databases/ui/db-panel.ts

```diff
@@ -67,13 +68,11 @@ export class DbPanel extends DisposableObject {
       return;
     }
 
-    if (this.dbManager.doesRemoteListExist(listName)) {
-      void showAndLogErrorMessage(
-        `A list with the name '${listName}' already exists`,
-      );
-    } else {
-      await this.dbManager.addNewRemoteList(listName);
-    }
+    const highlightedItem = await this.getHighlightedDbItem();
+
+    const listKind = highlightedItem?.kind || DbItemKind.RootRemote;
```

Comment:
Worth adding a comment here to explain this decision and what we expect to see in the future.